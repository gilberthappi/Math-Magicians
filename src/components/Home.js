const Home = () => (
  <section className="hero">
    <h1>Welcome to our Page!</h1>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore non
      mollitia officia itaque iusto aspernatur doloremque magni voluptate
      distinctio excepturi. Libero assumenda sed a, ex error tenetur accusamus
      repudiandae, saepe enim optio reiciendis est id reprehenderit mollitia,
      ipsum sunt corporis?
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore non
      mollitia officia itaque iusto aspernatur doloremque magni voluptate
      distinctio excepturi. Libero assumenda sed a, ex error tenetur accusamus
      repudiandae, saepe enim optio reiciendis est id reprehenderit mollitia,
      ipsum sunt corporis?
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore non
      mollitia officia itaque iusto aspernatur doloremque magni voluptate
      distinctio excepturi. Libero assumenda sed a, ex error tenetur accusamus
      repudiandae, saepe enim optio reiciendis est id reprehenderit mollitia,
      ipsum sunt corporis?
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore non
      mollitia officia itaque iusto aspernatur doloremque magni voluptate
      distinctio excepturi. Libero assumenda sed a, ex error tenetur accusamus
      repudiandae, saepe enim optio reiciendis est id reprehenderit mollitia,
      ipsum sunt corporis?
    </p>
  </section>
);

export default Home;
