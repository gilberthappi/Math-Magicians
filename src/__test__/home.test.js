import renderer from 'react-test-renderer';
import Home from '../components/Home';

describe('Home components testing', () => {
  it('Renders correctly', () => {
    const tree = renderer
      .create(<Home />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
