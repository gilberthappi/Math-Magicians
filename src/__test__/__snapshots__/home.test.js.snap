// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home components testing Renders correctly 1`] = `
<section
  className="hero"
>
  <h1>
    Welcome to our Page!
  </h1>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore non mollitia officia itaque iusto aspernatur doloremque magni voluptate distinctio excepturi. Libero assumenda sed a, ex error tenetur accusamus repudiandae, saepe enim optio reiciendis est id reprehenderit mollitia, ipsum sunt corporis? Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore non mollitia officia itaque iusto aspernatur doloremque magni voluptate distinctio excepturi. Libero assumenda sed a, ex error tenetur accusamus repudiandae, saepe enim optio reiciendis est id reprehenderit mollitia, ipsum sunt corporis?
  </p>
  <p>
    Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore non mollitia officia itaque iusto aspernatur doloremque magni voluptate distinctio excepturi. Libero assumenda sed a, ex error tenetur accusamus repudiandae, saepe enim optio reiciendis est id reprehenderit mollitia, ipsum sunt corporis? Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore non mollitia officia itaque iusto aspernatur doloremque magni voluptate distinctio excepturi. Libero assumenda sed a, ex error tenetur accusamus repudiandae, saepe enim optio reiciendis est id reprehenderit mollitia, ipsum sunt corporis?
  </p>
</section>
`;
